parser.rb

```diff
@@ -0,0 +1,43 @@
+
+class Parser
+  def initialize(filename)
+    @file = File.open(filename, "r")
+    @text = ''
+    @file.each_line do |txt|
+      @text = @text + txt
+    end
+  end
+
+  def array_of_words
+    @text.split(/\W+/)
```

Comment:
This is super-simple, and it works pretty well.  Some trouble with contractions ("it's"), but maybe that's unavoidable without a lot of complexity.